Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -99,8 +99,8 @@ configuration.
     helm install cilium |CHART_RELEASE| \\
         --namespace kube-system \\
         --set kubeProxyReplacement=strict \\
-        --set k8sServiceHost=API_SERVER_IP \\
-        --set k8sServicePort=API_SERVER_PORT
+        --set k8sServiceHost=$API_SERVER_IP \\
+        --set k8sServicePort=$API_SERVER_PORT
```

Comment:
I'm more or less okay with this. The reason we did not add the `$` in front was that we've seen users not reading the doc properly and then wondering why cilium is stuck connecting to kube-apiserver. This happened during quick copy'n'paste of the commands (or expecting them to be magically defined in the env already) where empty `$API_SERVER_IP` / `$API_SERVER_PORT` won't throw an error whereas `API_SERVER_IP` / `API_SERVER_PORT` will and force user to read doc more closely. @brb thoughts?

Reply:
I see the rationale behind this decision and would be fine with leaving it as is.

However, in the `helm` command concerning XDP NodePort on EKS, we even specify the commands to define these variables one section above. So at least in that case it could be misleading for the user to see `API_SERVER_IP`/`API_SERVER_PORT`.

Reply:
if `API_SERVER_IP` is unclear, maybe we can make it more clear with something like `REPLACE_WITH_API_SERVER_IP`?

Reply:
late to the party, but `$MY_API_SERVER_{API,PORT}` might be a different option.

Reply:
`REPLACE_WITH_API_SERVER_IP` SGTM.

Reply:
Alright, so I'd change it to `REPLACE_WITH_API_SERVER_IP`/`REPLACE_WITH_API_SERVER_PORT` except for the case in the EKS guide, where I'd use `$API_SERVER_IP`/`$API_SERVER_PORT` since we explicitly define them in the section above the `helm` command. Everybody fine with this approach?